pyaccel/lifetime.py

```diff
@@ -4,12 +4,14 @@
 import importlib as _implib
 from copy import deepcopy as _dcopy
 import numpy as _np
+from collections import namedtuple as _namedtuple
```

Comment:
There's a function from `mathphys` called `get_namedtuple` that you could use here:

```
from mathphys.functions import get_namedtuple
OPTICS = get_namedtuple('Optics', ['EdwardsTeng', 'Twiss'])
```

Reply:
thanks! done.